### PR TITLE
added::  Meta Tags,  Sitemap, and  robots.txt for SEO Improvement

### PR DESCRIPTION
**Pull request by myselfshivams**
Issue #5

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -6,7 +6,27 @@ const inter = Inter({ subsets: ["latin"] });
...
+  keywords: "URL shortener, shorten URLs, free URL shortener",
+  openGraph: {
+    title: "tturl",
+    description: "Shorten URLs in one click",
+    url: "https://tturl.netlify.app",
+    type: "website",
+    siteName: "tturl",
+    images: [
+      {
+        url: "https://tturl.netlify.app/og-image.jpg",
```
Comment (muruga21) [inline app/layout.tsx:19]: use the url from env file.. if i gonna get a domain.. its not necessary to change it in code right

File: next-sitemap.config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    siteUrl: 'https://tturl.netlify.app',
```
Comment (muruga21) [inline next-sitemap.config.js:2]: use the URL from env

**Review by muruga21 — CHANGES_REQUESTED** [review]

**Comment by muruga21** [comment]
@myselfshivams great work man.. i have requested some changes.. can you do that ?

**Comment by muruga21** [comment]
i have merged another pr.. which has favicon.. make sure you dont have conflicts

thanks

File: next-sitemap.config.js (hunk 2 above)
Comment (myselfshivams) [reply next-sitemap.config.js]: > use the URL from env

Where is the env ?

File: app/layout.tsx (hunk 1 above)
Comment (myselfshivams) [reply app/layout.tsx]: > use the url from env file.. if i gonna get a domain.. its not necessary to change it in code right

Could you please provide me url, I will change it

**Comment by myselfshivams** [comment]
> @myselfshivams great work man.. i have requested some changes.. can you do that ?



> i have merged another pr.. which has favicon.. make sure you dont have conflicts
> 
> thanks

Yeah, It have conflict but I will fix it and do the changes as per your need.

**Pull request closed without merging**